src/main/webapp/app/exercises/file-upload/assess/file-upload-assessment-dashboard.component.html

```diff
@@ -53,7 +53,9 @@ <h4 jhiTranslate="artemisApp.fileUploadAssessment.dashboard.heading" [translateV
                         </ng-container>
                     </td>
                     <td>
-                        <span *ngIf="submission.result && submission.result.score != null">{{ submission.result.score }} %</span>
+                        <span *ngIf="getLatestSubmissionResult(submission) && getLatestSubmissionResult(submission)!.score != null"
```

Comment:
It would be better to save the `result` in a global variable instead of using the method in the check. See also other places where you are doing this that way (e.g. line 57)

Reply:
I agree, in the html we should avoid method calls

Reply:
The refactoring of the function calls within the HTML will be done in a followup PR next week!